core/src/main/java/org/apache/accumulo/core/spi/scan/HintScanPrioritizer.java

```diff
@@ -80,6 +81,8 @@ private static int getPriority(ScanInfo si, int defaultPriority, HintProblemActi
     String prio = si.getExecutionHints().get("priority");
     if (prio != null) {
       try {
+        ScanUserDataLogger.logTrace(null, si.getUserData(),
```

Comment:
Why are we passing `null` for the logger? Under what circumstances would we want to pass `null` vs. something else?